tests/test_checker/test_noqa.py

```diff
@@ -33,7 +33,7 @@
     'WPS110': 3,
     'WPS111': 1,
     'WPS112': 1,
-    'WPS113': 1,
+    'WPS113': 0,
```

Comment:
So, what happened here? Why this was changed?

Reply:
Since now `WPS113` is raised only when `i_control_code = False`, the error doesn't occur during `test_noqa_fixture_disabled` test run. Maybe we should have separate list of violations which only occur when `i_control_code` = False? Like WPS411, for example.
Or maybe I don't understand or missed something?

Reply:
`i_control_code = False` list would be awesome to have! Will you please open a new issue for that?

Reply:
Done!
#957